app/views/questions/_option_set_modal.html.erb

```diff
@@ -0,0 +1,20 @@
+<%# new option set modal %>
+<% if form_mode != :show && can?(:create, OptionSet) %>
+  <div class="new_option_set_form_wrapper" title="<%= I18n.t('activerecord.action_links.option_set.new') %>">
```

Comment:
Change single quotes to double in this file plz?

Reply:
done